integration-tests/aqua/examples/streamArgs.aqua

```diff
@@ -15,4 +17,40 @@ func retrieve_records(peer: string) -> [][]string:
 
 func modify_stream(stream: *string) -> []string:
   stream <<- "appended value"
-  <- stream
\ No newline at end of file
+  <- stream
+
+func useStream(stream: *string) -> *string:
+    stream <<- "valueUseStream"
+    <- stream
+
+func returnStream() -> *string:
+    r <- useStream(*[])
+    r <<- "valueReturnStream"
+    <- r
+
+func lng280Bug() -> []string:
+    stream <- returnStream()
+    stream <<- "valueTop"
+    <- stream
+
+func lng280BugWithFor() -> []string:
+    results: *string
+    arr = [1,2,3]
+
+    for a <- arr:
+      stream <- returnStream()
+      results <<- stream[0]
+      results <<- stream[1]
+
+    <- results
+
+service StreamService("test-service"):
+    store(numbers: []u32, n: u32)
+
+func callService(stream: *u32, n: u32):
+    StreamService.store(stream, n)
+
+func lng280BugWithForAnonStream():
+  arr = [1,2,3,4,5]
+  for a <- arr:
+    callService(*[], a)
```

Comment:
I don't understand, what does this code test?
Stream is always empty so it is not tested if it is restricted correctly in `for`